tox.ini

```diff
@@ -79,14 +82,16 @@ extras =
 commands =
     pip freeze --all --no-input
     oldestdeps: python -c "import astropy.time; astropy.time.update_leap_seconds()"
-    !online-!hypothesis-!figure: {env:PYTEST_COMMAND} {posargs}
-    online: {env:PYTEST_COMMAND} --hypothesis-show-statistics --reruns 2 --reruns-delay 15 --timeout=180 --remote-data=any {posargs}
```

Comment:
Probably, I'll put this back

Reply:
Actually I've changed my mind - this will just re-run *any* test that fails, which if a bunch of stuff fails for non-remote reasons is a waste of test time. I'm going to leave this out, and if there are specific remote tests that need retrying we can mark them as such in the future.